app/views/travel_advice/index.html.erb

```diff
@@ -28,18 +28,19 @@
             <div class="travel-advice-notice__content">
               <p class="govuk-body">
                 <strong>
-                  Under current UK COVID-19 restrictions, you must stay at home (or in Wales, stay local). It is illegal to travel abroad for holidays.
+                  It is illegal to travel abroad from the UK for holidays.
                 </strong>
-                Check the <a href="/guidance/national-lockdown-stay-at-home">rules in England</a>, <a href="https://www.gov.scot/coronavirus-covid-19/">Scotland</a>, <a href="https://gov.wales/covid-19-alert-levels">Wales</a> and <a href="https://www.nidirect.gov.uk/articles/coronavirus-covid-19-regulations-guidance-what-restrictions-mean-you">Northern Ireland</a>.
+                Follow current COVID-19 rules where you live: <a href="/guidance/national-lockdown-stay-at-home">England</a>, <a href="https://www.gov.scot/coronavirus-covid-19/">Scotland</a>, <a href="https://gov.wales/covid-19-alert-levels">Wales</a> and <a href="https://www.nidirect.gov.uk/articles/coronavirus-covid-19-regulations-guidance-what-restrictions-mean-you">Northern Ireland</a>.
               </p>
               <p class="govuk-body">
-                Do not travel unless you have a legally permitted reason to do so. In England, you must <a href="/guidance/coronavirus-covid-19-declaration-form-for-international-travel">complete a declaration form for international travel</a> (except for travel to Ireland).
+                In England, you must have a <a href="/guidance/coronavirus-covid-19-declaration-form-for-international-travel">permitted reason to travel abroad</a> and complete the declaration form.
```

Comment:
I think this supposed to be on the same line as "Follow current COVID-19 rules where you live..."